### PR TITLE
Feature/01: Header content

### PR DESCRIPTION
**Pull request by liebeckm**

### CODE REVIEW

File: index-01.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (alexWaligorski) [inline index-01.html]: Suggestion: Change title to more specific one?

File: index-01.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#experience">Experience</a>
+        <a href="#skills">Skills</a>
+        <a href="education">Education</a>
```
Comment (alexWaligorski) [inline index-01.html]: hashtag missing in href

File: index-01.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      </nav>
+    </header>
+    <main>
+      <article id="#experience"></article>
```
Comment (alexWaligorski) [inline index-01.html]: id shouldn't include hashtag

File: index-01.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+      <article id="#skills"></article>
```
Comment (alexWaligorski) [inline index-01.html]: id shouldn't include hashtag

**Review by alexWaligorski — COMMENTED** [review]
Nicely done! Just some minor changes needed. Good night and sleep well!

File: index-01.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
Comment (PReichetanz) [inline index-01.html]: We don't have to name an html file with numbers, so you can safely change this one to `index.html`.

**Pull request merged**